README.md

```diff
@@ -13,42 +13,54 @@ With MEV-Boost, validators can access blocks from a marketplace of builders. Bui
 
 ## How does MEV-Boost work?
 
-
-PoS node operators must run three pieces of software: a validator client, consensus client, and an execution client. MEV-boost is a sidecar for the Consensus Client, a separate piece of open source software, which queries and outsources block-building to a network of builders. Block builders prepare full blocks, optimizing for MEV extraction and fair distribution of rewards. They then submit their blocks to relays.
+PoS node operators must run three pieces of software: a Validator client, a Consensus client, and an Execution client. MEV-boost is a sidecar for the Consensus client, a separate piece of open source software, which queries and outsources block-building to a network of builders. Block builders prepare full blocks, optimizing for MEV extraction and fair distribution of rewards. They then submit their blocks to relays.
 
 Relays aggregate blocks from **multiple** builders in order to select the block with the highest fees. One instance of MEV-boost can be configured by a validator to connect to **multiple** relays. The Consensus Layer client of a validator proposes the most profitable block received from MEV-boost to the Ethereum network for attestation and block inclusion.
 
+A MEV-Boost security assessment was conducted on 2022-06-20 by [lotusbumi](https://github.com/lotusbumi). Additional audits of surrounding infrastructure, such as the Flashbots Relay, are currently underway.
+
 
-![mev-boost service integration overview](https://raw.githubusercontent.com/flashbots/mev-boost/main/docs/mev-boost-integration-overview.png)
+![MEV-Boost service integration overview](https://raw.githubusercontent.com/flashbots/mev-boost/main/docs/mev-boost-integration-overview.png)
 
 ## Who can run MEV-Boost?
 
 MEV-Boost is a piece of software that any PoS Ethereum node operator (including solo validators) can run as part of their Beacon Client software. It is compatible with any Ethereum consensus client. Support and installation instructions for each client can be found [here](#installing).
 
-
 ---
 
 See also:
 
 * [boost.flashbots.net](https://boost.flashbots.net)
-* [mev-boost Docker images](https://hub.docker.com/r/flashbots/mev-boost)
+* [MEV-Boost Docker images](https://hub.docker.com/r/flashbots/mev-boost)
 * [wiki](https://github.com/flashbots/mev-boost/wiki) & [troubleshooting guide](https://github.com/flashbots/mev-boost/wiki/Troubleshooting)
-* [mev-boost relay source code](https://github.com/flashbots/mev-boost-relay)
+* [MEV-Boost relay source code](https://github.com/flashbots/mev-boost-relay)
 * Specs:
   * [Builder API](https://ethereum.github.io/builder-specs)
   * [Flashbots Relay API](https://flashbots.notion.site/Relay-API-Spec-5fb0819366954962bc02e81cb33840f5)
 
 
 # Table of Contents
 
+- [Table of Contents](#table-of-contents)
 - [Background](#background)
 - [Installing](#installing)
+  - [Binaries](#binaries)
+  - [From source](#from-source)
+    - [`go install`](#go-install)
```

Comment:
remove 3rd level ToC

Reply:
I cannot unless we reformat the ### markdown in the development sections, so I removed ### markdown.